### PR TITLE
Use generated types

### PR DESCRIPTION
**Pull request by ishrak-imam**
Use the generated type defs in resolvers.

### CODE REVIEW

File: subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import { Context } from '../../types';
 import { getCallParams } from '../../utils/call';
 import { notEmpty } from '../../utils/notEmpty';
+import type {DemocracySummary, Democracy} from '../../generated/resolvers-types'
 
 export const democracySummary = async (
-  parent: { address?: string },
-  args: { address?: string },
+  _: { address?: string },
```
Comment (franjs) [inline subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts:7]: If these are not used let's change them to something like `Record<string, never>`, that `address` creates confusion

File: subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts [hunk 2]
```diff
@@ -26,10 +27,10 @@ export const democracySummary = async (
 };
 
 export const democracy = async (
-  parent: { address?: string },
-  args: { address?: string },
+  _: { address?: string },
+  __: { address?: string },
```
Comment (franjs) [inline subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts:31]: Same here

**Review by franjs — APPROVED** [review]

File: subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts (hunk 1 above)
Comment (ishrak-imam) [reply subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts:7]: Fixed here https://github.com/litentry/api-gateway/pull/77/commits/cff81e31d5d3ddc7261ce3a571ba7c3c8f7883ab

File: subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts (hunk 2 above)
Comment (ishrak-imam) [reply subschemas/substrate-api-proxy/src/resolvers/Query/democracy.ts:31]: Fixed here https://github.com/litentry/api-gateway/pull/77/commits/cff81e31d5d3ddc7261ce3a571ba7c3c8f7883ab

**Pull request merged**